src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,46 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    this.hidden = undefined;
```

Comment:
why undefined?

Reply:
and why you introduce hidden here? It's not a property of Animal in overall, just Herbivore